README.md

```diff
@@ -32,6 +32,32 @@ jobs:
 
 Included in this repo is a [basic workflow](.github/workflows/plugins-ci.yml) for use across the majority of plugins, as well as variants with service containers.
 
+### Enable workflow Linter job
+
+By setting the `lint` value to `true` when using the [basic workflow](.github/workflows/plugins-ci.yml) the CI will first run the linter job.
+
+__Example:__ running the linter job first with the [basic workflow](.github/workflows/plugins-ci.yml)
+
+```yml
+name: CI
+
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+      - '*.md'
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - '*.md'
+
+jobs:
+  call-reuseable-workflow:
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v2
+    with:
+      lint: true
```

Comment:
```suggestion
      lint: false # default `true`
```

Reply:
The example needs updating to remove the `lint` input usage, as it's no longer in place in this PR.

Reply:
added back the lint input in the last iteration.